tests/testthat/test-scale.r

```diff
@@ -19,3 +19,24 @@ test_that("NA.value works for discrete", {
   expect_that(dscale(x, pal, "grey50")[1], equals("grey50"))
   expect_that(dscale(x, pal, "grey50")[5], equals("grey50"))
 })
+
+test_that("train_continuous maintains class", {
+  expect_equal(train_continuous(1:5),
+               c(1, 5),
+               info = "Normal numeric input works with train_continuous")
```

Comment:
Please remove all uses of `info`

Reply:
I can do that, but why remove it?  I find it to be useful to understand the underlying rationale for the test.  Is there a preferred way to indicate rationale?

Reply:
Rational should usually be encoded in the test name, or in a comment. In this case, I don't think that "Normal numeric input works with train_continuous" adds anything to my understanding of the test - it's implied by the test name that you'll be iterating through different types, and "working" is implied by `expect_equal()`